pom.xml

```diff
@@ -43,6 +43,10 @@
       <groupId>io.github.openfeign</groupId>
       <artifactId>feign-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Limitations in `appmap-java` mean that classes on the boot classpath can't currently be instrumented. One of the updates I made to `appmap-java` captures requests made using Apache's HttpClient. This change and the change to `application.properties` tells Feign to use it.